apps/choleskey/choleskey_stdpar.cpp

```diff
@@ -0,0 +1,78 @@
+// Cholesky Decomposition: stdpar
+#include "argparse/argparse.hpp"
+#include "commons.hpp"
+
+#include <algorithm>
+#include <execution>  // Include the execution header
+#include <experimental/mdspan>
+#include <iostream>
+#include <numeric>
+#include <vector>
+
+using namespace std;
+
+struct Multiply {
+  int operator()(int a, int b) const { return a * b; }
+};
```

Comment:
nit: prefer to have it as lambda function. Also, it seems it can be generalized and used by line 39 (the n*n).

Reply:
sure. done